docs/ossm/ossm2-migration/multi-tenancy/README.md

```diff
@@ -110,19 +110,28 @@ oc create ns istio-system-tenant-a
 
    Here we are setting `discoverySelectors` on our `Istio` resource. In 3.0, controlplanes by default watch the entire cluster and when managing multiple controlplanes on a single cluster, you must narrow the scope of each controlplane by setting `discoverySelectors`. In this example, we use the label `tenant` but you can use any label or combination of labels that you choose.
 
+   > **_NOTE:_** For simplicity, we are using a minimal example for Istio resource. Read [SMCP to Istio mapping](#TODO) to see how to transform fields used in SMCP resource to fields in Istio resource.
+
    ```yaml
    apiVersion: sailoperator.io/v1alpha1
    kind: Istio
    metadata:
      name: istio-tenant-a
    spec:
      namespace: istio-system-tenant-a
+     version: v1.23.0
```

Comment:
Should this be `1.24.1` now? Maybe when version aliases is released we can update these examples to `v1.24-latest` or something.

Reply:
Yes, this version will be annoying. We have https://issues.redhat.com/browse/OSSM-8698 to fix it before GA because we will have to fix it in many places, not just Istio examples but also all those example outputs e.g. for istioctl ps